src/components/markdown-renderer/hooks/use-trimmed-content.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { useSelector } from 'react-redux'
+import { ApplicationState } from '../../../redux'
+import { useMemo } from 'react'
+
+export const useTrimmedContent = (content: string): [trimmedContent: string, isTrimmed: boolean] => {
```

Comment:
It's non-functional, but maybe change the variable name in the method type signature as well?

Reply:
forgot that. fixed